src/context.cpp

```diff
@@ -13,17 +13,79 @@
 #include <librealsense2/rs.h>              // RS2_API_FULL_VERSION_STR
 #include <src/librealsense-exception.h>
 
+#include <rsutils/os/special-folder.h>
+#include <rsutils/os/executable-name.h>
 #include <rsutils/easylogging/easyloggingpp.h>
 #include <rsutils/string/from.h>
 #include <rsutils/json.h>
 using json = nlohmann::json;
 
+#include <fstream>
+
+
+namespace librealsense {
+
+
+    static nlohmann::json load_config()
```

Comment:
I think it might be useful to have a `load/save_json( filename )` utility. I know that Noa has a task concerning saving and parsing json content.

Reply:
Not quite sure why, it's very straightforward as you see...